feedback.md

```diff
@@ -1,30 +1,51 @@
 # JS Adagrams
-## What We're Looking For
-
-<!--
-
-Comprehension questions: They are all subjective!
-
-Notes for grading: Optional Wave 5 asks students to convert the syntax to a class, so you will see differences in code structure between submissions.
-
- -->
-
-Feature | Feedback
---- | ---
-**General** |
-Answered comprehension questions |
-Small commits with meaningful commit messages |
-**Code Requirements** |
-`drawLetters` method |
-Uses appropriate data structure to store the letter distribution |
-All tests for `drawLetters` pass |
-`usesAvailableLetters` method |
-All tests for `usesAvailableLetters` pass |
-`scoreWord` method |
-Uses appropriate data structure to store the letter scores |
-All tests for `scoreWord` pass |
-**Optional** |
-`highestScoreFrom` method |
-Appropriately handles edge cases for tie-breaking logic |
-All tests for `highestScoreFrom` pass |
-**Overall** |
+
+<!-- Instructors: The checkmarks are already there, so just delete them for any line items that aren't met. -->
+
+## Major Learning Goals/Code Review
+
+<!-- Instructors: Feel free to practice creating specific feedback by referencing a line of code if you'd like. For example, you may say something like "nice custom method in `calculator.rb` line 42." This is optional. -->
+
+| Criteria | yes/no, and optionally any details/lines of code to reference |
+| --- | --- |
+| Correctly uses variables, and only uses `const` and `let` variables. The program prefers `const` variables. The program never uses `var`. | ✔️?
```

Comment:
Not really having to do with the code review, but we'll want to address that we want them to default to `const` before they start this.

Reply:
This is a good point! I wonder if we can schedule in a quick something about that over the next few days.